package.json

```diff
@@ -97,6 +98,7 @@
     "execa": "8.0.1",
     "express": "4.18.3",
     "install": "^0.13.0",
+    "lodash": "^4.17.21",
```

Comment:
Is there no smaller dependency? Or just few lines of helper code?

At the very least you should use lodash-es